tests/unit/command/test_metrics.py

```diff
@@ -226,9 +226,6 @@ def test_metrics_show_with_valid_falsey_values():
         all_branches=True,
     ) == textwrap.dedent(
         """\
```

Comment:
Since the new format is more advanced, we need to add some more unit tests like this one. E.g. multiple Revisions, No revision, multiple metrics, precision. E.g. see `diff` tests above.

Reply:
Ill add more tests